docs/samples/README.md

```diff
@@ -0,0 +1,45 @@
+## KFServing Examples
+
+### KFServing InferenceService with out of the box Predictor
+[Deploy SKLearn Model with out-of-the-box InferenceService](./docs/samples/sklearn)
```

Comment:
IMO simplify these to just "SKLearn", "ONNX Model with ONNX Runtime", etc. Deploy and InferenceService is stated in the header.